strongback/docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Houston docker file
+# Builds an ubuntu base with liftoff for packaging debian applications
```

Comment:
Also worth noting that docker will probably be involved down the line with automated testing.